### PR TITLE
Feature/oe 181 setter dashboard

### PR DESCRIPTION
**Pull request by Ron-Junior**
## Goal

Get customers when logged user is a setter

## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

File: resources/views/livewire/customer/edit.blade.php [hunk 1]
```diff
@@ -36,13 +36,13 @@ class="inline-flex items-center border-b-2 border-green-base hover:border-green-
                                           name="customer.date_of_sale" :value="$customer->date_of_sale"/>
                     </div>
 
-                    <div class="col-span-1 md:col-span-2">
+                    <div class="col-span-1 md:col-span-2 @if($customer->setter_id == user()->id) md:col-span-3 @endif">
```
Comment (iammayron) [inline resources/views/livewire/customer/edit.blade.php:39]: You need to place the `md:col-span-2` in a @else conditional

**Review by iammayron — CHANGES_REQUESTED** [review]

**Review by iammayron — APPROVED** [review]
